src/peep/staff.c

```diff
@@ -253,7 +256,7 @@ void game_command_hire_new_staff_member(int* eax, int* ebx, int* ecx, int* edx,
 		newPeep->pathfind_goal.z = 0xFF;
 		newPeep->pathfind_goal.direction = 0xFF;
 
-		uint8 colour = gStaffColours[staff_type > 2 ? 2 : staff_type];
+		uint8 colour = gStaffColours[clamp(0, staff_type, STAFF_TYPE_SECURITY)];
```

Comment:
You still need to replace `0` with `STAFF_TYPE_HANDYMAN` in the clamp function -- 0 is also a magic constant here.